content/helpers.go

```diff
@@ -211,7 +212,7 @@ func seekReader(r io.Reader, offset, size int64) (io.Reader, error) {
 	if ok {
 		nn, err := seeker.Seek(offset, io.SeekStart)
 		if nn != offset {
-			return nil, errors.Wrapf(err, "failed to seek to offset %v", offset)
+			return nil, fmt.Errorf("failed to seek to offset %v: %w", offset, err)
```

Comment:
This is a weird case where `err` shouldn't be nil if it is, the previous behavior was wrong and now it will give an odd message